src/web-api.js

```diff
@@ -375,6 +375,75 @@ class WebApi {
       .get_job_status(partner_params.user_id, partner_params.job_id, options);
   }
 
+	get_web_token(requestParams) {
+		return new Promise((resolve, reject) => {
+			if (!requestParams) {
+				reject( new Error('Please ensure that you send through request params') );
+			}
+
+			if (typeof requestParams !== 'object') {
+				reject( new Error('Request params needs to be an object') );
+			}
+
+			['user_id', 'job_id', 'product'].forEach(requiredParam => {
+				if (!requestParams[requiredParam]) {
+					reject( new Error(`${requiredParam} is required to get a web token`) );
+				}
+			});
+
+			const timestamp = new Date().toISOString();
+			const signature = new Signature(this.partner_id, this.api_key).generate_signature(timestamp).signature;
+
+			const body = JSON.stringify({
```

Comment:
request is missing `callback_url`, which is a required field.

Reply:
fixed.